### ISSUE TITLE
Fix duplicated model artifact names on W&B

### ISSUE THREAD

**Issue by aazuspan** [lemma-osu/naip-cnn#28]
While migrating models for #27, I noticed that the model artifacts stored with the last 7 runs in W&B are considered [different versions of the same artifact named "model"](https://wandb.ai/aazuspan-team/naip-cnn/artifacts/model/model), whereas previous runs each their own distinct model artifact. This was inadvertently introduced in https://github.com/lemma-osu/naip-cnn/pull/23/commits/04107872a7a61535b12e457c3870400e0cc96d4b, when I overrode the default artifact name assigned to the model because it was too long and causing errors. I had assumed that artifacts belonged to runs so that each run would have an artifact named "model", but instead it seems that artifacts belong to the project and are just *linked* to the run that logged them, so we now have one artifact "model" with multiple runs logging to it.

Treating every run as a different version of the same model would probably make sense if we were just tweaking hyperparameters, but since different runs represent different training datasets and labels, I think they should maintain distinct artifacts. That should make it easier to track and version models, especially when I run the migration and each model gets a new `2.16.2` tag.

To fix this, we need to:

1. Retroactively re-save those last 8 models to distinct artifacts associated with their runs.
2. In the future, log each model with a run-specific name. Probably `f"{run.name}_model"`, e.g. `brisk-rain-129_model`.

**Comment by aazuspan** [comment]
Step 1 is complete. There was definitely a better way to do this, but for future reference I just ended up with a manual process where I ran the code below individually for each of the 7 runs, swapping in their IDs and model versions. It's not elegant, but it worked, and each run now has its own run artifact. I deleted the shared "model" artifact to avoid confusion as those would otherwise be identified as the "latest" model for these runs.

```python
from pathlib import Path

import wandb


if __name__ == "__main__":
    RUN_ID = "einhgdj8"
    MODEL_VERSION = "v6"

    with wandb.init(
        entity="aazuspan-team",
        project="naip-cnn",
        id=RUN_ID, 
        resume="allow") as run:

        # Select the model artifact
        latest_model = run.use_artifact(f"model:{MODEL_VERSION}")
        # Download the model file
        model_artifact = list(Path(latest_model.download()).rglob("*.keras"))[0].as_posix()
        # Re-log it as a new artifact with the run-specific name
        run.log_artifact(model_artifact, name=f"{run.name}_model", type="model")
        run.finish()   
```
